### PR TITLE
Backport TextMesh and expose dynamic font vector outlines

### PR DESCRIPTION
**Pull request by bruvzg**
3.x backport of #60507 and #44289 and #61283.
Screenshot 2022-05-23 at 09 31 41

### CODE REVIEW

File: COPYRIGHT.txt [hunk 1]
```diff
@@ -316,6 +316,12 @@ Comment: Minimal PCG32 implementation
 Copyright: 2014, M.E. O'Neill
 License: Apache-2.0
 
+Files: ./thirdparty/misc/polypartition.cpp
```
Comment (akien-mga) [inline COPYRIGHT.txt:319]: I think we already have this, it's named `triangulator.cpp`.

Comment (bruvzg) [reply COPYRIGHT.txt:319]: Indeed, it seems to be the same. I'll check if it works with existing version.

Comment (bruvzg) [reply COPYRIGHT.txt:319]: Existing one seems to be working fine, not sure why it was renamed (and which version is it, README have `Version: TBD, class was renamed`).

**Review by akien-mga — APPROVED** [review]

File: COPYRIGHT.txt (hunk 1 above)
Comment (akien-mga) [reply COPYRIGHT.txt]: This was updated in `master` with https://github.com/godotengine/godot/pull/45125 but we didn't backport it to `3.x`.

**Pull request merged**